app/code/Magento/InventoryExportStock/etc/module.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0"?>
+<!--
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+-->
+<config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:noNamespaceSchemaLocation="urn:magento:framework:Module/etc/module.xsd">
+    <module name="Magento_InventoryExportStock" setup_version="1.0.0">
+        <sequence>
+            <module name="Magento_Catalog"/>
+            <module name="Magento_InventoryApi"/>
+            <module name="Magento_InventoryCatalogApi"/>
+            <module name="Magento_InventoryExportStockApi"/>
+            <module name="Magento_InventorySalesApi"/>
+        </sequence>
```

Comment:
do we really need the sequence here? looks excessive

Reply:
sequence used for module load order....